modules/dynamic-plugins/proc-using-topology-plugin.adoc

```diff
@@ -1,20 +1,39 @@
 = Using the Topology plugin
+
 Topology is a front-end plugin that enables you to view the workloads as nodes that power any service on the Kubernetes cluster.
 
+== Enabling users to use the Topology plugin
```

Comment:
Modularization: a procedure cannot contain 2 procedures. 

* Split each procedure into a dedicated file.
* Aggregate the 2 procedures in an assembly file.